packages/pds/src/labeler/hive.ts

```diff
@@ -0,0 +1,118 @@
+import stream from 'stream'
+import axios from 'axios'
+import FormData from 'form-data'
+import { Labeler } from './base'
+import Database from '../db'
+import { BlobStore } from '@atproto/repo'
+import { keywordLabeling } from './util'
+
+const HIVE_ENDPOINT = 'https://api.thehive.ai/api/v2/task/sync'
```

Comment:
Do they have a sandbox environment we should be using in dev?

Reply:
don't think so :/